src/runtime-rs/crates/runtimes/virt_container/src/container_manager/process.rs

```diff
@@ -133,22 +133,14 @@ impl Process {
         let io_name = io_name.to_string();
         let logger = self.logger.new(o!("io_name" => io_name));
         let _ = tokio::spawn(async move {
-            loop {
-                match tokio::io::copy(&mut reader, &mut writer).await {
-                    Err(e) => {
-                        if let Some(error_code) = e.raw_os_error() {
-                            if error_code == libc::EAGAIN {
-                                continue;
-                            }
-                        }
-                        warn!(logger, "run_io_copy: failed to copy stream: {}", e);
-                    }
-                    Ok(length) => {
-                        warn!(logger, "run_io_copy: stop to copy stream length {}", length)
-                    }
-                };
-                break;
-            }
+            match tokio::io::copy(&mut reader, &mut writer).await {
+                Err(e) => {
+                    warn!(logger, "run_io_copy: failed to copy stream: {}", e);
+                }
+                Ok(length) => {
+                    warn!(logger, "run_io_copy: stop to copy stream length {}", length)
```

Comment:
I think the stream copy future terminates on this. Maybe the log could be at `debug` level or `info` level, what do you think.

Reply:
Thanks, @Ji-Xinyou change the `warn!()` to `info!()`.